Directory.Build.targets

```diff
@@ -33,6 +34,28 @@
     <_RuntimeIdentifiers Include="$(RuntimeIdentifiers)" />
   </ItemGroup>
 
+  <Target Name="_GetSymbolsForSnupkg" DependsOnTargets="_WalkEachTargetPerFramework" Inputs="%(_RuntimeIdentifiers.Identity)" Outputs="_unused">
+    <MSBuild
+      Condition="'$(IncludeRIDSpecificBuildOutput)' == 'true'"
+      Projects="$(MSBuildProjectFullPath)"
+      Targets="_GetDebugSymbolsWithTfm"
+      Properties="RuntimeIdentifier=%(_RuntimeIdentifiers.Identity)">
+
+      <Output
+          TaskParameter="TargetOutputs"
+          ItemName="_DebugSymbolsWithTfm" />
+    </MSBuild>
+    <PropertyGroup>
+      <!-- Need to set this back to True so that the snupkg will contain symbols. -->
+      <IncludeBuildOutput>true</IncludeBuildOutput>
+    </PropertyGroup>
+    <ItemGroup>
+      <_TargetPathsToSymbols Include="@(_DebugSymbolsWithTfm)">
```

Comment:
Could you have instead just added these to the `RIDSpecificPackageFile` item?

Reply:
I believe that wouldn't work because if I did that then they would be packaged in the .nupkg and not in the .snupkg. `_TargetPathsToSymbols` is the Item that gets populated with the pdbs that you want to go to the .snupkg but not the .nupkg. Another annoying thing is that with this Item, you can't really specify a `PackagePath`, so basically this item only hast two metadatas that will decide where will the pdb be placed in the .snupkg. Pdbs will be placed at lib/%(_TargetPathsToSymbols.TargetFramework)/%(_TargetPathsToSymbols.Identity) and there doesn't seem to be  a way of overriding this. That is why for now I'm using the hack of appending the RID into the TargetFramework on the next line, so that the Windows and Linux pdbs won't collide.

Reply:
This is pretty ugly, can you file an issue on nuget to enable a better story?

Reply:
Will do.